_scripts/test_linux.sh

```diff
@@ -76,3 +76,11 @@ else
 	exit $x
 fi
 
+export GOARCH=ppc64le
+go run _scripts/make.go --tags exp.linuxppc64le
```

Comment:
There's a bunch of test failures in CI, I think this might be what's causing them.

Reply:
I think it was the TestGeneratedDoc. It should be fix now

Reply:
The tests now pass, this line however is never executed because of the exit above.